server/data/licenceApiClient.ts

```diff
@@ -251,6 +252,7 @@ export default class LicenceApiClient extends RestClient {
     )) as ComReviewCount
   }
 
+  // TODO: don't believe these two are used now we have moved the caseload to the backend - remove these from where they are used and the tests?
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏Something to think about